README.md

```diff
@@ -15,29 +15,28 @@ pkg> add Lattices
 ## A glance of the interface
 
 ```julia-repl
-julia> using Lattice
-
-julia> l = chain(4)
-Chain Lattice:
-  Periodic boundary
-  size: (4,)
-
-julia> for each in sites(l)
-          # do something on sites
-          @show each
-       end
-
-each = 1
-each = 2
-each = 3
-each = 4
-
-julia> for (a, b) in edges(l, length=2)
-          # do something on edges
-
-          println("this is point a: ", a)
-          println("this is point b: ", b)
-       end
+julia> using Lattices               
+                                    
+julia> l = Chain(4)                 
+Chain Lattice                       
+boundary: Periodic                  
+size:     (4,)                      
+                                    
+julia> for each in sites(l)         
+           @show each               
+       end                          
+each = 1                            
+each = 2                            
+each = 3                            
+each = 4                            
+                                    
+julia> for (from, to) in edges(l)   
```

Comment:
I don't think the lattice sites have a direction by default? Or should we assume so?

Reply:
No, you're right. We should replace from and to with I and j and println the arrow as <->

I'm on my phone. Feel free to change it. Otherwise I'll do so later/tomorrow.